xbmc/video/VideoInfoTag.h

```diff
@@ -367,6 +367,7 @@ class CVideoInfoTag : public IArchivable, public ISerializable, public ISortable
     std::string title; //!< Title of the movie set
     int id; //!< ID of movie set in database
     std::string overview; //!< Overview/description of the movie set
+    std::string originalTitle; //<! Original title of the movie set
```

Comment:
It could. As I'm still getting used to this I try and change as little as possible (unknown unknowns / unintended consequences and all that) - but I can certainly have a go.

Reply:
Lower priority, as a separate commit or PR.

Reply:
OK. Will leave for now.

Reply:
Done